api-reference/endpoint/get-software.mdx

```diff
@@ -0,0 +1,108 @@
+---
+title: "Get Software Info"
+api: "POST https://eol.xeol.com/api/v1/query"
+description: "This endpoint gets a software's support information"
+---
+
+### Body
+
+<ParamField body="identifier" type="string[]" required="true">
+  List of software identifiers.
+  
+  Can be either in `cpe`, `purl`, or `custom` format.
+  
+  If `custom`, just use the software's name like this: `Red Hat Enterprise Linux`
+</ParamField>
+
+### Response
+
+<ResponseField name="results" type="object[]">
+  List of software with their listed supports
+
+  <Expandable title="result">
+    <ResponseField name="name" type="string">
+      Name of the matched artifact
+    </ResponseField>
+
+    <ResponseField name="version" type="string">
+      Release cycle of the matched artifact
+    </ResponseField>
+
+    <ResponseField name="type" type="string">
+      Type of the matched artifact, either `software` or `package` or `os` or `product`
+    </ResponseField>
+
+    <ResponseField name="released" type="datetime">
+      Release date of the matched artifact
+    </ResponseField>
+
+    <ResponseField name="lts" type="boolean">
+      Whether the matched artifact has `Long Term Support`
+    </ResponseField>
+
+    <ResponseField name="support" type="object[]">
+      All availabe support including `Extended Lifecycle Support` for matched artifact
+      
+      This will always have at least **one** item.
+      <Expandable title="support">
+        <ResponseField name="id" type="UUID">
+          Unique support ID 
+        </ResponseField>
+
+        <ResponseField name="name" type="string">
+          Official support name
+        </ResponseField>
+
+        <ResponseField name="eol" type="datetime">
+          End-of-life date
+        </ResponseField>
+      </Expandable>
+    </ResponseField>
+  </Expandable>
+</ResponseField>
+
+<RequestExample>
+  ```bash curl
+  curl --location --request POST 'https://eol.xeol.com/api/v1/query' \
+  --header 'Content-Type: application/json' \
+  --header 'Authorization: <XEOL_API_KEY>' \
+  --data-raw '{
+    "identifiers": [
+      "cpe:/o:redhat:enterprise_linux:6",
+    ]
+  }'
+  ```
+</RequestExample>
+
+<ResponseExample>
+  ```json 200
+  {
+    "results": [
+      {
+        "name": "Red Hat Enterprise Linux",
+        "version": "6",
+        "type": "os",
+        "released": "2010-11-09 00:00:00",
+        "lts": false,
+        "support": [
+          {
+            "id": "0fcc7bfd-759b-4a9f-a062-87c73149ee86"
```

Comment:
the IDs here should ideally be semantic instead of a UUID so that they're easy to identify.

Think of an example where a user have a config file for their support versions. It may look something like this

.support.yml
```
entities:
    name: Red Hat Enterprise Linux
    support_id: SUPPORT-RHEL-L0
```